archetype/engine-v2/src/test/resources/schema/archetype.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<archetype-script xmlns="https://helidon.io/archetype/2.0"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="https://helidon.io/archetype/2.0 file:/archetype.xsd">
+    <context>
+        <boolean path="test.option1">true</boolean>
+        <list path="test.array1">
+            <value>hello</value>
+        </list>
+        <enum path="test.enum1">
+            <value>bob</value> <!-- error, this is not one of the enum values -->
```

Comment:
The comment refers to the the input and the fact that `bob` is not a listed value of the enum. Let's change this to be a valid value since the validations are not implemented yet.